maxmindupdater/__init__.py

```diff
@@ -0,0 +1,74 @@
+import hashlib
+import os
+import shutil
+import sys
+import tarfile
+
+import maxminddb
+import requests
+
+__doc__ = 'Function to keep a maxmind database file up to date'
+__version__ = '0.1.0'
+__url__ = 'https://github.com/yola/maxmind-updater'
+
+
+def _hash_file(filename):
+    if not os.path.exists(filename):
+        return ''
+
+    block_size = 65536
+    hasher = hashlib.md5()
+
+    with open(filename, 'rb') as f:
+        buf = f.read(block_size)
+        while len(buf) > 0:
```

Comment:
I think this feature of python is not well-known enough to be worth using. Many readers would be forced to look it up. I've used an alternative solution which is DRY and easily readable.